qiskit/visualization/state_visualization.py

```diff
@@ -548,7 +548,12 @@ def plot_state_paulivec(
 ):
     """Plot the paulivec representation of a quantum state.
 
-    Plot a bargraph of the mixed state rho over the pauli matrices
+    Plot a bargraph of the density matrix of a quantum state using as a basis all
+    possible tensor products of Pauli operators and identities, that is,
+    :math:`\\{\\bigotimes_{i=0}^{N-1}P_i\\}_{P_i\\in \\{I,X,Y,Z\\}}`, where
```

Comment:
FYI If you annotate the string as raw string, you can use normal Latex here without needeing the double `{` 🙂 like
```python
r"""...

:math:`\{\bigotimes ...`
"""

Reply:
I didn't know about that. Thank you very much Julien 😄 

Changed to raw in [e63ff2e](https://github.com/Qiskit/qiskit-terra/pull/8420/commits/e63ff2e04563a5d16081d91286b6125363e82bc0)